server/src/test/java/io/crate/integrationtests/AlterTableIntegrationTest.java

```diff
@@ -59,4 +65,103 @@ public void test_alter_table_drop_column_dropped_meanwhile() {
         assertSQLError(() -> execute(plan).getResult())
             .hasMessageContaining("Column b unknown");
     }
+
+
+    @Test
+    public void test_alter_table_drop_simple_column() {
+        execute("create table t(id integer primary key, a integer, b integer)");
+        execute("insert into t(id, a, b) values(1, 11, 111)");
+        execute("refresh table t");
+        execute("select * from t");
```

Comment:
if I understand correctly `SourceParser` already filters out dropped columns. Could you add assertions to the new tests that `SELECT _raw` doesn't contain dropped columns (dropped subtrees)?

Reply:
See: https://github.com/crate/crate/pull/14622#discussion_r1305705582